app/src/main/res/values-night/colors.xml

```diff
@@ -72,6 +72,7 @@
     <color name="tab_tray_item_thumbnail_background_normal_theme">@color/tab_tray_item_thumbnail_background_dark_theme</color>
     <color name="tab_tray_item_thumbnail_icon_normal_theme">@color/tab_tray_item_thumbnail_icon_dark_theme</color>
     <color name="tab_tray_selected_mask_normal_theme">@color/tab_tray_selected_mask_dark_theme</color>
+    <color name="top_sites_border_color">@color/top_sites_border_color_dark</color>
```

Comment:
Can you move this with the other Top sit colors around Line 80. Let's not map it to `@color/top_sites_border_color_dark` because it is unnecessary to have this indirection.


We can just do
```suggestion
    <color name="top_sites_border_color">@color/dark_grey_90</color>
```